### DISCUSSION TITLE
Effect of Grid size on ammonia liquid release from the nozzle

### DISCUSSION THREAD

**Discussion by cqc330** [firemodels/fds#13823] [Q&A] [answered]
Hi, I want to simulate a case of ammonia horizontal jet release using fds 6.9.1, so I am using a NOZZLE to spray liquid ammonia. However, when I use a 1m grid, the sprayangle does not match what I expect. But when I use a 0.2m grid, the spray angle is more consistent with my command settings. I would like to know why the grid size causes such a significant difference in the nozzle behavior.

Additionally, I would like to know whether my NOZZLE settings are correct or not. Furthermore, if I want to simulate a large-scale ammonia horizontal jet release using non-uniform grid system, the results in the larger grid area significantly differs from the small grid area? Implementing uniform grids (especially smaller grids) for such a large domain size (880m x 660m x 40m) is not practically feasible.  I request for your thoughtful suggestions. Thank you!
( I have 140cores cluster can be used)

[1.0m_grid.txt](https://github.com/user-attachments/files/17987923/1.0m_grid.txt)
[0.2m_grid.txt](https://github.com/user-attachments/files/17987925/0.2m_grid.txt)

![1 0m](https://github.com/user-attachments/assets/debcff67-6318-472d-b32d-21480db18622)
![0 2m](https://github.com/user-attachments/assets/21d276f8-dcad-418d-be84-7a332cda0754)

**Comment by drjfloyd** [comment, marked as answer]
Droplets impose drag on the gas, and a 44 um droplet is going to quickly exchange its momentum with the gas and then just move with the local gas velocty. On a large grid, drops injected to the left will add left and away from the nozzle momentum. Drops injected to the right will add right and away from the nozzle momentum. The net momentum change to the gas is the sum over all of the droplets. Left and right in the same grid cell will cancel leaving only the away from the nozzle. At 1 m you are not resolving the momentum exchange.

Your inputs have the correct syntax as you are getting droplets injected. We cannot tell you if the values you have chosen are correct. We do not know the details of your scenario to judge if your orfice size, leak rate, etc. are appropriate. Those are deciscions that you need to make and justify as the modeler.  However, you have a 2 cm orifice and a flow of 369 lpm. This to me does not seem to be conditions where I would expect 44 um droplets.

**Reply by cqc330** [reply]
I truly appreciate your detailed answer. It helped a lot!
